components/molecules/Auth/style.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+const StyledSignUp = styled.div`
+    display:flex;
+`; 
+
+const StyledLink = styled.a`
+    font-family: "Rockwell Regular";
```

Comment:
Please move the font family to constant file and use it here through styled component theme.